web/src/main/java/nl/tehdreamteam/se42/web/soap/conversation/SoapConversationEndpoint.java

```diff
@@ -14,7 +14,7 @@
  * @author Oscar de Leeuw
  */
 @WebService(targetNamespace = SoapWebServiceConstants.DEFAULT_NAMESPACE + "conversation")
-public class SoapConversationController implements ConversationController {
+public class SoapConversationEndpoint implements ConversationEndpoint {
     @Override
     public void addMessage(@WebParam(name = "token") String token, @WebParam(name = "content") String content, @WebParam(name = "conversation_id") long conversationId) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonarqube.tehmessiah.nl/coding_rules#rule_key=squid%3AS1186)